extensions/binding/org.eclipse.smarthome.binding.meteoblue/src/main/java/org/eclipse/smarthome/binding/meteoblue/internal/MeteoBlueHandlerFactory.java

```diff
@@ -34,7 +38,8 @@
 @Component(service = ThingHandlerFactory.class, immediate = true)
 public class MeteoBlueHandlerFactory extends BaseThingHandlerFactory {
 
-    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Collections.singleton(THING_TYPE_WEATHER);
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Sets.union(
```

Comment:
Should be possible to implement without an external library:
```
Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Stream.concat(
                    BRIDGE_THING_TYPES_UIDS.stream(),
                    SUPPORTED_THING_TYPES_UIDS.stream()).collect(toSet());
```

Reply:
Would be nice if you remove the external dependency here.

Reply:
It'd be nice if your suggested code compiled, but it doesn't.
I tried changing it to this, but it also doesn't compile:

````
Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS =
        Stream.concat(
        BRIDGE_THING_TYPES_UIDS.stream(),
        MeteoBlueBindingConstants.SUPPORTED_THING_TYPES_UIDS.stream()).collect(toSet());
````

````
[ERROR] MeteoBlueBindingConstants.SUPPORTED_THING_TYPES_UIDS.stream()).collect(toSet());
[ERROR] ^^^^^
[ERROR] The method toSet() is undefined for the type MeteoBlueHandlerFactory
````

Reply:
Upps sorry, I missed the `Collectors.` in front of `toSet()`.

Reply:
Yes, that appears to work.
Done.